1739724976097	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1739727450635	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1739727450636	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1739727450640	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1739727450649	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1739727450653	Sync.SyncScheduler	DEBUG	Woke from sleep.
1739727450657	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1739727450657	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1739727450659	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1739727452799	Sync.Service	DEBUG	User-Agent: Zen/1.7.6b (Linux x86_64) FxSync/1.137.0.20250207111303.desktop
1739727452799	Sync.Service	INFO	Starting sync at 2025-02-16 18:37:32 in browser session L9DNyWVvsAlj
1739727452804	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1739727452805	Sync.Status	INFO	Resetting Status.
1739727452805	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1739727452806	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1739727452806	FirefoxAccounts	DEBUG	getOAuthToken enter
1739727452806	FirefoxAccounts	TRACE	getCachedToken returning cached token
1739727452806	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1739727452806	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1739727452807	Sync.SyncAuthManager	INFO	Getting sync key
1739727452807	FirefoxAccounts	DEBUG	getOAuthToken enter
1739727452807	FirefoxAccounts	TRACE	getCachedToken returning cached token
1739727452807	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1739727452807	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1739727452807	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1739727452807	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1739727452807	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1739727452823	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1739727452823	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1739727452824	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1739727452824	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1739727452825	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1739727452825	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1739727452825	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1739727452825	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1739727452827	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1739727452827	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1739727452830	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/266102437/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1739727452831	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1739727452831	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1739727452832	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1739727452832	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1739727452833	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1739727452841	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
